src/Sparrow/Platform/Win32/Win32MemoryMethods.cs

```diff
@@ -1,9 +1,10 @@
 ﻿using System;
 using System.Runtime.InteropServices;
+using System.Security;
 
 namespace Sparrow.Platform.Win32
 {
-    public static unsafe class Win32MemoryProtectMethods
+    public static unsafe class Win32MemoryMethods
```

Comment:
Because it was a mess, all methods there are Memory methods, not protect methods. They happen to be useful for that, but we have many things scattered all over the place.

Reply:
Break it to a separate PR, it make it really hard to review the changes for this PR, which are already complex enough to require careful review